openedx/features/smart_referral/models.py

```diff
@@ -7,6 +10,9 @@
 
 
 class SmartReferral(TimeStampedModel):
+    """
+    Model contains fields related to Smart Referral
+    """
     user = models.ForeignKey(User, related_name='smart_referrals', related_query_name='smart_referral',
                              on_delete=models.CASCADE)
```

Comment:
Please add a blank line after class docstring as decided internally.

Similarly add blank line in the following

1. test_api_views.py after line 32 and 122
2. test_helpers.py  after line 28